<!DOCTYPE html>
<html lang="en">

    <head>
    <title>What Can Article Level Metrics Do for You?</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="author" content="" />
    <meta name="keywords" content="example, markdown, article-level metrics, reproducibility" />

    <!-- RDFa Metadata (in DublinCore) -->
    <meta property="dc:title" content="What Can Article Level Metrics Do for You?" />
    <meta property="dc:creator" content="" />
    <meta property="dc:format" content="text/html" />
    <meta property="dc:language" content="en" />
    <meta property="dc:identifier" content="/2013/12/11/what-can-article-level-metrics-do-for-you" />
    <meta property="dc:rights" content="CC-BY-NC" />
    <meta property="dc:source" content="Gobbledygook" />
    <meta property="dc:subject" content="Scholarly Communication" />
    <meta property="dc:type" content="website" />

    <!-- Google Scholar Metadata -->
    <meta name="citation_author" content=""/>
    <meta name="citation_title" content="What Can Article Level Metrics Do for You?"/>
    <meta name="citation_journal_title" content="Gobbledygook"/>

    <link rel="alternate" type="application/rss+xml" title="Gobbledygook" href="/rss.xml" />
    <link rel="alternate" type="application/atom+xml" title="Gobbledygook" href="/atom.xml" />

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- CSS -->
    <link rel="stylesheet" href="/assets/app-7b11536ae5581522c573f3f553590ae2.css">
    <link rel="stylesheet" href="/assets/jasny-bootstrap.min-29c6b484ddca96fa14a48394c0cf415b.css">
    <link rel="stylesheet" href="/assets/navmenu-push-f57c8aea8e44efd42b5d7fc575383151.css">
    <link rel="stylesheet" href="/assets/fluidbox-cc1b5668a8c598b1c2fcc5e15e073f33.css">

    <!-- d3.js -->
    <script src="//cdnjs.cloudflare.com/ajax/libs/d3/3.4.2/d3.min.js"></script>

    <!-- Typekit -->
    <script type="text/javascript" src="//use.typekit.net/kil7puw.js"></script>
    <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
  </head>


  <body>
    <nav id="navmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation">
  <a class="navmenu-brand" href="#">Gobbledygook</a>
  <ul class="nav navmenu-nav">
  
    <li><span class="glyphicon glyphicon-time"></span> <a href="http://github.com/mfenner/blog/commits/source/_posts/2013-12-11-what-can-article-level-metrics-do-for-you.Rmd">Version History</a></li>
    <li><span class="glyphicon glyphicon-edit"></span> <a href="http://prose.io/#mfenner/mfenner.github.io/edit/source/_posts/2013-12-11-what-can-article-level-metrics-do-for-you.Rmd">Send Edit Request</a></li>
    
    <li><span class="glyphicon glyphicon-tags"></span> Tags
      <ul class="tag-list">
        
        
  
  	<li><a href="/tags.html#example-ref">example</a></li>
  
  	<li><a href="/tags.html#markdown-ref">markdown</a></li>
  
  	<li><a href="/tags.html#article-level metrics-ref">article-level metrics</a></li>
  
  	<li><a href="/tags.html#reproducibility-ref">reproducibility</a></li>
  



      </ul>
    </li>
    
    <li><span class="glyphicon glyphicon-home"></span> <a href="/">Home</a></li>
    <li><span class="glyphicon glyphicon-info-sign"></span> <a href="/about.html">About</a></li>
    <li><span class="glyphicon glyphicon-tags"></span> <a href="/tags.html">All Tags</a></li>
  </ul>
</nav>


    <div class="navbar navbar-default navbar-fixed-top">
      <button type="button" class="navbar-toggle" data-toggle="offcanvas" data-target="#navmenu" data-canvas="body">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>

    <div class="container">
      <div class="row">
        <div class="col-md-8 col-md-offset-2">
          <div class="page-header">
            <h1>What Can Article Level Metrics Do for You?</h1>
            <p>11 December 2013 by
            
            <a href="http://orcid.org/0000-0003-1419-2405" title="">Martin Fenner</a>
            </p>
          </div>
        </div>
      </div>

      <div class="content">
        <div class="row">
          <div class="col-md-8 col-md-offset-2">
            <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body>
<p><em>Article-level metrics (ALMs) provide a wide range of metrics about the uptake of an individual journal article by the scientific community after publication. They include citations, usage statistics, discussions in online comments and social media, social bookmarking, and recommendations. In this essay, we describe why article-level metrics are an important extension of traditional citation-based journal metrics and provide a number of example from ALM data collected for PLOS Biology.</em></p>
<p><em>This is an open-access article distributed under the terms of the Creative Commons Attribution License, authored by me and <a href="http://dx.doi.org/10.1371/journal.pbio.1001687">originally published Oct 22, 2013 in PLOS Biology</a>.</em></p>
<p>The scientific impact of a particular piece of research is reflected in how this work is taken up by the scientific community. The first systematic approach that was used to assess impact, based on the technology available at the time, was to track citations and aggregate them by journal. This strategy is not only no longer necessary â€” since now we can easily track citations for individual articles â€” but also, and more importantly, journal-based metrics are now considered a poor performance measure for individual articles <span class="citation" data-cites="Campbell2008 Glanzel2013">(Campbell, 2008; Glänzel &amp; Wouters, 2013)</span>. One major problem with journal-based metrics is the variation in citations per article, which means that a small percentage of articles can skew, and are responsible for, the majority of the journal-based citation impact factor, as shown by Campbell <span class="citation" data-cites="Campbell2008">(2008)</span> for the 2004 <em>Nature</em> Journal Impact Factor. <strong>Figure 1</strong> further illustrates this point, showing the wide distribution of citation counts between <em>PLOS Biology</em> research articles published in 2010. <em>PLOS Biology</em> research articles published in 2010 have been cited a median 19 times to date in Scopus, but 10% of them have been cited 50 or more times, and two articles <span class="citation" data-cites="Narendra:2010fw Dickson:2010ix">(Dickson, Wang, Krantz, Hakonarson, &amp; Goldstein, 2010; Narendra et al., 2010)</span> more than 300 times. <em>PLOS Biology</em> metrics are used as examples throughout this essay, and the dataset is available in the supporting information (<strong>Data S1</strong>). Similar data are available for an increasing number of other publications and organizations.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co"># code for figure 1: density plots for citation counts for PLOS Biology</span>
<span class="co"># articles published in 2010</span>

<span class="co"># load May 20, 2013 ALM report</span>
alm &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">"data/alm_report_plos_biology_2013-05-20.csv"</span>, <span class="dt">stringsAsFactors =</span> <span class="ot">FALSE</span>)

<span class="co"># only look at research articles</span>
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, alm$article_type ==<span class="st"> "Research Article"</span>)

<span class="co"># only look at papers published in 2010</span>
alm$publication_date &lt;-<span class="st"> </span><span class="kw">as.Date</span>(alm$publication_date)
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, alm$publication_date &gt;<span class="st"> "2010-01-01"</span> &amp;<span class="st"> </span>alm$publication_date &lt;=
<span class="st">    "2010-12-31"</span>)

<span class="co"># labels</span>
colnames &lt;-<span class="st"> </span><span class="kw">dimnames</span>(alm)[[<span class="dv">2</span>]]
plos.color &lt;-<span class="st"> "#1ebd21"</span>
plos.source &lt;-<span class="st"> "scopus"</span>

plos.xlab &lt;-<span class="st"> "Scopus Citations"</span>
plos.ylab &lt;-<span class="st"> "Probability"</span>

quantile &lt;-<span class="st"> </span><span class="kw">quantile</span>(alm[, plos.source], <span class="kw">c</span>(<span class="fl">0.1</span>, <span class="fl">0.5</span>, <span class="fl">0.9</span>), <span class="dt">na.rm =</span> <span class="ot">TRUE</span>)

<span class="co"># plot the chart</span>
opar &lt;-<span class="st"> </span><span class="kw">par</span>(<span class="dt">mai =</span> <span class="kw">c</span>(<span class="fl">0.5</span>, <span class="fl">0.75</span>, <span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="dt">omi =</span> <span class="kw">c</span>(<span class="fl">0.25</span>, <span class="fl">0.1</span>, <span class="fl">0.25</span>, <span class="fl">0.1</span>), <span class="dt">mgp =</span> <span class="kw">c</span>(<span class="dv">3</span>,
    <span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="dt">fg =</span> <span class="st">"black"</span>, <span class="dt">cex.main =</span> <span class="dv">2</span>, <span class="dt">cex.lab =</span> <span class="fl">1.5</span>, <span class="dt">col =</span> plos.color,
    <span class="dt">col.main =</span> plos.color, <span class="dt">col.lab =</span> plos.color, <span class="dt">xaxs =</span> <span class="st">"i"</span>, <span class="dt">yaxs =</span> <span class="st">"i"</span>)

d &lt;-<span class="st"> </span><span class="kw">density</span>(alm[, plos.source], <span class="dt">from =</span> <span class="dv">0</span>, <span class="dt">to =</span> <span class="dv">100</span>)
d$x &lt;-<span class="st"> </span><span class="kw">append</span>(d$x, <span class="dv">0</span>)
d$y &lt;-<span class="st"> </span><span class="kw">append</span>(d$y, <span class="dv">0</span>)
<span class="kw">plot</span>(d, <span class="dt">type =</span> <span class="st">"n"</span>, <span class="dt">main =</span> <span class="ot">NA</span>, <span class="dt">xlab =</span> <span class="ot">NA</span>, <span class="dt">ylab =</span> <span class="ot">NA</span>, <span class="dt">xlim =</span> <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="dt">frame.plot =</span> <span class="ot">FALSE</span>)
<span class="kw">polygon</span>(d, <span class="dt">col =</span> plos.color, <span class="dt">border =</span> <span class="ot">NA</span>)
<span class="kw">mtext</span>(plos.xlab, <span class="dt">side =</span> <span class="dv">1</span>, <span class="dt">col =</span> plos.color, <span class="dt">cex =</span> <span class="fl">1.25</span>, <span class="dt">outer =</span> <span class="ot">TRUE</span>, <span class="dt">adj =</span> <span class="dv">1</span>,
    <span class="dt">at =</span> <span class="dv">1</span>)
<span class="kw">mtext</span>(plos.ylab, <span class="dt">side =</span> <span class="dv">2</span>, <span class="dt">col =</span> plos.color, <span class="dt">cex =</span> <span class="fl">1.25</span>, <span class="dt">outer =</span> <span class="ot">TRUE</span>, <span class="dt">adj =</span> <span class="dv">0</span>,
    <span class="dt">at =</span> <span class="dv">1</span>, <span class="dt">las =</span> <span class="dv">1</span>)

<span class="kw">par</span>(opar)</code></pre></div>
<figure>
<a href=""><img src="/images/2013-12-11_figure_1.svg" alt="Figure 1. Citation counts for PLOS Biology articles published in 2010. Scopus citation counts plotted as a probability distribution for all 197 PLOS Biology research articles published in 2010. Data collected May 20, 2013. Median 19 citations; 10% of papers have at least 50 citations."></a><figcaption><strong>Figure 1. Citation counts for PLOS Biology articles published in 2010.</strong> Scopus citation counts plotted as a probability distribution for all 197 <em>PLOS Biology</em> research articles published in 2010. Data collected May 20, 2013. Median 19 citations; 10% of papers have at least 50 citations.</figcaption>
</figure>
<p>Scientific impact is a multi-dimensional construct that can not be adequately measured by any single indicator <span class="citation" data-cites="Glanzel2013 bollenEtal2009 Schekman2013">(Bollen, Sompel, Hagberg, &amp; Chute, 2009; Glänzel &amp; Wouters, 2013; Schekman &amp; Patterson, 2013)</span>. To this end, PLOS has collected and displayed a variety of metrics for all its articles since 2009. The array of different categorised article-level metrics (ALMs) used and provided by PLOS as of August 2013 are shown in <strong>Figure 2</strong>. In addition to citations and usage statistics, i.e., how often an article has been viewed and downloaded, PLOS also collects metrics about: how often an article has been saved in online reference managers, such as Mendeley; how often an article has been discussed in its comments section online, and also in science blogs or in social media; and how often an article has been recommended by other scientists. These additional metrics provide valuable information that we would miss if we only consider citations. Two important shortcomings of citation-based metrics are that (1) they take years to accumulate and (2) citation analysis is not always the best indicator of impact in more practical fields, such as clinical medicine <span class="citation" data-cites="VanEck2013">(Eck, Waltman, Raan, Klautz, &amp; Peul, 2013)</span>. Usage statistics often better reflect the impact of work in more practical fields, and they also sometimes better highlight articles of general interest (for example, the 2006 <em>PLOS Biology</em> article on the citation advantage of Open Access articles <span class="citation" data-cites="Eysenbach2006">(Eysenbach, 2006)</span>, one of the 10 most-viewed articles published in <em>PLOS Biology</em>).</p>
<figure>
<a href=""><img src="/images/2013-12-11_figure_2.png" alt="Figure 2. Article-level metrics used by PLOS in August 2013 and their categories. Taken from (Lin &amp; Fenner, 2013) with permission by the authors."></a><figcaption><strong>Figure 2. Article-level metrics used by PLOS in August 2013 and their categories.</strong> Taken from <span class="citation" data-cites="Lin2013">(Lin &amp; Fenner, 2013)</span> with permission by the authors.</figcaption>
</figure>
<p>A bubble chart showing all 2010 <em>PLOS Biology</em> articles (<strong>Figure 3</strong>) gives a good overview of the year’s views and citations, plus it shows the influence that the article type (as indicated by dot color) has on an article’s performance as measured by these metrics. The weekly <em>PLOS Biology</em> publication schedule is reflected in this figure, with articles published on the same day present in a vertical line. <strong>Figure 3</strong> also shows that the two most highly cited 2010 <em>PLOS Biology</em> research articles are also among the most viewed (indicated by the red arrows), but overall there isn’t a strong correlation between citations and views. The most-viewed article published in 2010 in <em>PLOS Biology</em> is an essay on Darwinian selection in robots <span class="citation" data-cites="Floreano2010">(Floreano &amp; Keller, 2010)</span>. Detailed usage statistics also allow speculatulation about the different ways that readers access and make use of published literature; some articles are browsed or read online due to general interest while others that are downloaded (and perhaps also printed) may reflect the reader’s intention to look at the data and results in detail and to return to the article more than once.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co"># code for figure 3: Bubblechart views vs. citations for PLOS Biology</span>
<span class="co"># articles published in 2010.</span>

<span class="co"># Load required libraries</span>
<span class="kw">library</span>(plyr)

<span class="co"># load May 20, 2013 ALM report</span>
alm &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">"../data/alm_report_plos_biology_2013-05-20.csv"</span>, <span class="dt">stringsAsFactors =</span> <span class="ot">FALSE</span>,
    <span class="dt">na.strings =</span> <span class="kw">c</span>(<span class="st">"0"</span>))

<span class="co"># only look at papers published in 2010</span>
alm$publication_date &lt;-<span class="st"> </span><span class="kw">as.Date</span>(alm$publication_date)
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, alm$publication_date &gt;<span class="st"> "2010-01-01"</span> &amp;<span class="st"> </span>alm$publication_date &lt;=
<span class="st">    "2010-12-31"</span>)

<span class="co"># make sure counter values are numbers</span>
alm$counter_html &lt;-<span class="st"> </span><span class="kw">as.numeric</span>(alm$counter_html)

<span class="co"># lump all papers together that are not research articles</span>
reassignType &lt;-<span class="st"> </span>function(x) if (x ==<span class="st"> "Research Article"</span>) <span class="dv">1</span> else <span class="dv">0</span>
alm$article_group &lt;-<span class="st"> </span><span class="kw">aaply</span>(alm$article_type, <span class="dv">1</span>, reassignType)

<span class="co"># calculate article age in months</span>
alm$age_in_months &lt;-<span class="st"> </span>(<span class="kw">Sys.Date</span>() -<span class="st"> </span>alm$publication_date)/<span class="fl">365.25</span> *<span class="st"> </span><span class="dv">12</span>
start_age_in_months &lt;-<span class="st"> </span><span class="kw">floor</span>(<span class="kw">as.numeric</span>(<span class="kw">Sys.Date</span>() -<span class="st"> </span><span class="kw">as.Date</span>(<span class="kw">strptime</span>(<span class="st">"2010-12-31"</span>,
    <span class="dt">format =</span> <span class="st">"%Y-%m-%d"</span>)))/<span class="fl">365.25</span> *<span class="st"> </span><span class="dv">12</span>)

<span class="co"># chart variables</span>
x &lt;-<span class="st"> </span>alm$age_in_months
y &lt;-<span class="st"> </span>alm$counter
z &lt;-<span class="st"> </span>alm$scopus

xlab &lt;-<span class="st"> "Age in Months"</span>
ylab &lt;-<span class="st"> "Total Views"</span>

labels &lt;-<span class="st"> </span>alm$article_group
col.main &lt;-<span class="st"> "#1ebd21"</span>
col &lt;-<span class="st"> "#666358"</span>

<span class="co"># calculate bubble diameter</span>
z &lt;-<span class="st"> </span><span class="kw">sqrt</span>(z/pi)

<span class="co"># calculate bubble color</span>
getColor &lt;-<span class="st"> </span>function(x) <span class="kw">c</span>(<span class="st">"#c9c9c7"</span>, <span class="st">"#1ebd21"</span>)[x +<span class="st"> </span><span class="dv">1</span>]
colors &lt;-<span class="st"> </span><span class="kw">aaply</span>(labels, <span class="dv">1</span>, getColor)

<span class="co"># plot the chart</span>
opar &lt;-<span class="st"> </span><span class="kw">par</span>(<span class="dt">mai =</span> <span class="kw">c</span>(<span class="fl">0.5</span>, <span class="fl">0.75</span>, <span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="dt">omi =</span> <span class="kw">c</span>(<span class="fl">0.25</span>, <span class="fl">0.1</span>, <span class="fl">0.25</span>, <span class="fl">0.1</span>), <span class="dt">mgp =</span> <span class="kw">c</span>(<span class="dv">3</span>,
    <span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="dt">fg =</span> <span class="st">"black"</span>, <span class="dt">cex =</span> <span class="dv">1</span>, <span class="dt">cex.main =</span> <span class="dv">2</span>, <span class="dt">cex.lab =</span> <span class="fl">1.5</span>, <span class="dt">col =</span> <span class="st">"white"</span>,
    <span class="dt">col.main =</span> col.main, <span class="dt">col.lab =</span> col)

<span class="kw">plot</span>(x, y, <span class="dt">type =</span> <span class="st">"n"</span>, <span class="dt">xlim =</span> <span class="kw">c</span>(start_age_in_months, start_age_in_months +<span class="st"> </span><span class="dv">13</span>),
    <span class="dt">ylim =</span> <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">60000</span>), <span class="dt">xlab =</span> <span class="ot">NA</span>, <span class="dt">ylab =</span> <span class="ot">NA</span>, <span class="dt">las =</span> <span class="dv">1</span>)
<span class="kw">symbols</span>(x, y, <span class="dt">circles =</span> z, <span class="dt">inches =</span> <span class="kw">exp</span>(<span class="fl">1.3</span>)/<span class="dv">15</span>, <span class="dt">bg =</span> colors, <span class="dt">xlim =</span> <span class="kw">c</span>(start_age_in_months,
    start_age_in_months +<span class="st"> </span><span class="dv">13</span>), <span class="dt">ylim =</span> <span class="kw">c</span>(<span class="dv">0</span>, ymax), <span class="dt">xlab =</span> <span class="ot">NA</span>, <span class="dt">ylab =</span> <span class="ot">NA</span>, <span class="dt">las =</span> <span class="dv">1</span>,
    <span class="dt">add =</span> <span class="ot">TRUE</span>)
<span class="kw">mtext</span>(xlab, <span class="dt">side =</span> <span class="dv">1</span>, <span class="dt">col =</span> col.main, <span class="dt">cex =</span> <span class="fl">1.25</span>, <span class="dt">outer =</span> <span class="ot">TRUE</span>, <span class="dt">adj =</span> <span class="dv">1</span>, <span class="dt">at =</span> <span class="dv">1</span>)
<span class="kw">mtext</span>(ylab, <span class="dt">side =</span> <span class="dv">2</span>, <span class="dt">col =</span> col.main, <span class="dt">cex =</span> <span class="fl">1.25</span>, <span class="dt">outer =</span> <span class="ot">TRUE</span>, <span class="dt">adj =</span> <span class="dv">0</span>, <span class="dt">at =</span> <span class="dv">1</span>,
    <span class="dt">las =</span> <span class="dv">1</span>)

<span class="kw">par</span>(opar)</code></pre></div>
<figure>
<a href=""><img src="/images/2013-12-11_figure_3.svg" alt="Figure 3. Views vs. citations for PLOS Biology articles published in 2010. All 304 PLOS Biology articles published in 2010. Bubble size correlates with number of Scopus citations. Research articles are labeled green; all other articles are grey. Red arrows indicate the two most highly cited papers. Data collected May 20, 2013."></a><figcaption><strong>Figure 3. Views vs. citations for PLOS Biology articles published in 2010.</strong> All 304 <em>PLOS Biology</em> articles published in 2010. Bubble size correlates with number of Scopus citations. Research articles are labeled green; all other articles are grey. Red arrows indicate the two most highly cited papers. Data collected May 20, 2013.</figcaption>
</figure>
<p>When readers first see an interesting article, their response is often to view or download it. By contrast, a citation may be one of the last outcomes of their interest, occuring only about 1 in 300 times a PLOS paper is viewed online. A lot of things happen in between these potential responses, ranging from discussions in comments, social media, and blogs, to bookmarking, to linking from websites. These activities are usually subsumed under the term â€œaltmetrics,â€ and their variety can be overwhelming. Therefore, it helps to group them together into categories, and several organizations, including PLOS, are using the category labels of Viewed, Cited, Saved, Discussed, and Recommended (<strong>Figures 2 and 4</strong>, see also <span class="citation" data-cites="Lin2013">(Lin &amp; Fenner, 2013)</span>).</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co"># code for figure 4: bar plot for Article-level metrics for PLOS Biology</span>

<span class="co"># Load required libraries</span>
<span class="kw">library</span>(reshape2)

<span class="co"># load May 20, 2013 ALM report</span>
alm &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">"../data/alm_report_plos_biology_2013-05-20.csv"</span>, <span class="dt">stringsAsFactors =</span> <span class="ot">FALSE</span>,
    <span class="dt">na.strings =</span> <span class="kw">c</span>(<span class="dv">0</span>, <span class="st">"0"</span>))

<span class="co"># only look at research articles</span>
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, alm$article_type ==<span class="st"> "Research Article"</span>)

<span class="co"># make sure columns are in the right format</span>
alm$counter_html &lt;-<span class="st"> </span><span class="kw">as.numeric</span>(alm$counter_html)
alm$mendeley &lt;-<span class="st"> </span><span class="kw">as.numeric</span>(alm$mendeley)

<span class="co"># options</span>
plos.color &lt;-<span class="st"> "#1ebd21"</span>
plos.colors &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"#a17f78"</span>, <span class="st">"#ad9a27"</span>, <span class="st">"#ad9a27"</span>, <span class="st">"#ad9a27"</span>, <span class="st">"#ad9a27"</span>, <span class="st">"#ad9a27"</span>,
    <span class="st">"#dcebdd"</span>, <span class="st">"#dcebdd"</span>, <span class="st">"#789aa1"</span>, <span class="st">"#789aa1"</span>, <span class="st">"#789aa1"</span>, <span class="st">"#304345"</span>, <span class="st">"#304345"</span>)

<span class="co"># use subset of columns</span>
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, <span class="dt">select =</span> <span class="kw">c</span>(<span class="st">"f1000"</span>, <span class="st">"wikipedia"</span>, <span class="st">"researchblogging"</span>, <span class="st">"comments"</span>,
    <span class="st">"facebook"</span>, <span class="st">"twitter"</span>, <span class="st">"citeulike"</span>, <span class="st">"mendeley"</span>, <span class="st">"pubmed"</span>, <span class="st">"crossref"</span>, <span class="st">"scopus"</span>,
    <span class="st">"pmc_html"</span>, <span class="st">"counter_html"</span>))

<span class="co"># calculate percentage of values that are not missing (i.e. have a count of</span>
<span class="co"># at least 1)</span>
colSums &lt;-<span class="st"> </span><span class="kw">colSums</span>(!<span class="kw">is.na</span>(alm)) *<span class="st"> </span><span class="dv">100</span>/<span class="kw">length</span>(alm$counter_html)
exactSums &lt;-<span class="st"> </span><span class="kw">sum</span>(<span class="kw">as.numeric</span>(alm$pmc_html), <span class="dt">na.rm =</span> <span class="ot">TRUE</span>)

<span class="co"># plot the chart</span>
opar &lt;-<span class="st"> </span><span class="kw">par</span>(<span class="dt">mar =</span> <span class="kw">c</span>(<span class="fl">0.1</span>, <span class="fl">7.25</span>, <span class="fl">0.1</span>, <span class="fl">0.1</span>) +<span class="st"> </span><span class="fl">0.1</span>, <span class="dt">omi =</span> <span class="kw">c</span>(<span class="fl">0.1</span>, <span class="fl">0.25</span>, <span class="fl">0.1</span>, <span class="fl">0.1</span>),
    <span class="dt">col.main =</span> plos.color)

plos.names &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"F1000Prime"</span>, <span class="st">"Wikipedia"</span>, <span class="st">"Research Blogging"</span>, <span class="st">"PLOS Comments"</span>,
    <span class="st">"Facebook"</span>, <span class="st">"Twitter"</span>, <span class="st">"CiteULike"</span>, <span class="st">"Mendeley"</span>, <span class="st">"PubMed Citations"</span>, <span class="st">"CrossRef"</span>,
    <span class="st">"Scopus"</span>, <span class="st">"PMC HTML Views"</span>, <span class="st">"PLOS HTML Views"</span>)
y &lt;-<span class="st"> </span><span class="kw">barplot</span>(colSums, <span class="dt">horiz =</span> <span class="ot">TRUE</span>, <span class="dt">col =</span> plos.colors, <span class="dt">border =</span> <span class="ot">NA</span>, <span class="dt">xlab =</span> plos.names,
    <span class="dt">xlim =</span> <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">120</span>), <span class="dt">axes =</span> <span class="ot">FALSE</span>, <span class="dt">names.arg =</span> plos.names, <span class="dt">las =</span> <span class="dv">1</span>, <span class="dt">adj =</span> <span class="dv">0</span>)
<span class="kw">text</span>(colSums +<span class="st"> </span><span class="dv">6</span>, y, <span class="dt">labels =</span> <span class="kw">sprintf</span>(<span class="st">"%1.0f%%"</span>, colSums))

<span class="kw">par</span>(opar)</code></pre></div>
<figure>
<a href=""><img src="/images/2013-12-11_figure_4.svg" alt="Figure 4. Article-level metrics for PLOS Biology. Proportion of all 1,706 PLOS Biology research articles published up to May 20, 2013 mentioned by particular article-level metrics source. Colors indicate categories (Viewed, Cited, Saved, Discussed, Recommended), as used on the PLOS website."></a><figcaption><strong>Figure 4. Article-level metrics for PLOS Biology.</strong> Proportion of all 1,706 <em>PLOS Biology</em> research articles published up to May 20, 2013 mentioned by particular article-level metrics source. Colors indicate categories (Viewed, Cited, Saved, Discussed, Recommended), as used on the PLOS website.</figcaption>
</figure>
<p>All <em>PLOS Biology</em> articles are viewed and downloaded, and almost all of them (all research articles and nearly all front matter) will be cited sooner or later. Almost all of them will also be bookmarked in online reference managers, such as Mendeley, but the percentage of articles that are discussed online is much smaller. Some of these percentages are time dependent; the use of social media discussion platforms, such as Twitter and Facebook for example, has increased in recent years (93% of <em>PLOS Biology</em> research articles published since June 2012 have been discussed on Twitter, and 63% mentioned on Facebook). These are the locations where most of the online discussion around published articles currently seems to take place; the percentage of papers with comments on the PLOS website or that have science blog posts written about them is much smaller. Not all of this online discussion is about research articles, and perhaps, not surprisingly, the most-tweeted PLOS article overall (with more than 1,100 tweets) is a <em>PLOS Biology</em> perspective on the use of social media for scientists <span class="citation" data-cites="Bik2013">(Bik &amp; Goldstein, 2013)</span>.</p>
<p>Some metrics are not so much indicators of a broad online discussion, but rather focus on highlighting articles of particular interest. For example, science blogs allow a more detailed discussion of an article as compared to comments or tweets, and journals themselves sometimes choose to highlight a paper on their own blogs, allowing for a more digestible explanation of the science for the non-expert reader <span class="citation" data-cites="Fausto2012">(Fausto et al., 2012)</span>. Coverage by other bloggers also serves the same purpose; a good example of this is one recent post on the OpenHelix Blog <span class="citation" data-cites="Video2012">(“Video Tip of the Week: Turkeys and their genomes,” 2012)</span> that contains video footage of the second author of a 2010 <em>PLOS Biology</em> article <span class="citation" data-cites="Dalloul2010">(Dalloul et al., 2010)</span> discussing the turkey genome.</p>
<p>F1000Prime, a commercial service of recommendations by expert scientists, was added to the PLOS Article-Level Metrics in August 2013. We now highlight on the PLOS website when any articles have received at least one recommendation within F1000Prime. We also monitor when an article has been cited within the widely used modern-day online encyclopedia, Wikipedia. A good example of the latter is the Tasmanian devil Wikipedia page <span class="citation" data-cites="Tasmanian2013">(“Tasmanian devil,” 2013)</span> that links to a <em>PLOS Biology</em> research article published in 2010 <span class="citation" data-cites="Nilsson2010">(Nilsson et al., 2010)</span>. While a F1000Prime recommendation is a strong endorsement from peer(s) in the scientific community, being included in a Wikipedia page is akin to making it into a textbook about the subject area and being read by a much wider audience that goes beyond the scientific community.</p>
<p><em>PLOS Biology</em> is the PLOS journal with the highest percentage of articles recommended in F1000Prime and mentioned in Wikipedia, but there is only partial overlap between the two groups of articles because they focus on different audiences (<strong>Figure 5</strong>). These recommendations and mentions in turn show correlations with other metrics, but not simple ones; you can’t assume, for example, that highly cited articles are more likely to be recommended by F1000Prime, so it will be interesting to monitor these trends now that we include this information.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co"># code for figure 5: Venn diagram F1000 vs. Wikipedia for PLOS Biology</span>
<span class="co"># articles</span>

<span class="co"># load required libraries</span>
<span class="kw">library</span>(<span class="st">"plyr"</span>)
<span class="kw">library</span>(<span class="st">"VennDiagram"</span>)

<span class="co"># load May 20, 2013 ALM report</span>
alm &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">"../data/alm_report_plos_biology_2013-05-20.csv"</span>, <span class="dt">stringsAsFactors =</span> <span class="ot">FALSE</span>)

<span class="co"># only look at research articles</span>
alm &lt;-<span class="st"> </span><span class="kw">subset</span>(alm, alm$article_type ==<span class="st"> "Research Article"</span>)

<span class="co"># group articles based on values in Wikipedia and F1000</span>
reassignWikipedia &lt;-<span class="st"> </span>function(x) if (x &gt;<span class="st"> </span><span class="dv">0</span>) <span class="dv">1</span> else <span class="dv">0</span>
alm$wikipedia_bin &lt;-<span class="st"> </span><span class="kw">aaply</span>(alm$wikipedia, <span class="dv">1</span>, reassignWikipedia)
reassignF1000 &lt;-<span class="st"> </span>function(x) if (x &gt;<span class="st"> </span><span class="dv">0</span>) <span class="dv">2</span> else <span class="dv">0</span>
alm$f1000_bin &lt;-<span class="st"> </span><span class="kw">aaply</span>(alm$f1000, <span class="dv">1</span>, reassignF1000)
alm$article_group =<span class="st"> </span>alm$wikipedia_bin +<span class="st"> </span>alm$f1000_bin
reassignCombined &lt;-<span class="st"> </span>function(x) if (x ==<span class="st"> </span><span class="dv">3</span>) <span class="dv">1</span> else <span class="dv">0</span>
alm$combined_bin &lt;-<span class="st"> </span><span class="kw">aaply</span>(alm$article_group, <span class="dv">1</span>, reassignCombined)
reassignNo &lt;-<span class="st"> </span>function(x) if (x ==<span class="st"> </span><span class="dv">0</span>) <span class="dv">1</span> else <span class="dv">0</span>
alm$no_bin &lt;-<span class="st"> </span><span class="kw">aaply</span>(alm$article_group, <span class="dv">1</span>, reassignNo)

<span class="co"># remember to divide f1000_bin by 2, as this is the default value</span>
summary &lt;-<span class="st"> </span><span class="kw">colSums</span>(<span class="kw">subset</span>(alm, <span class="dt">select =</span> <span class="kw">c</span>(<span class="st">"wikipedia_bin"</span>, <span class="st">"f1000_bin"</span>, <span class="st">"combined_bin"</span>,
    <span class="st">"no_bin"</span>)), <span class="dt">na.rm =</span> <span class="ot">TRUE</span>)
rows &lt;-<span class="st"> </span><span class="kw">nrow</span>(alm)

<span class="co"># options</span>
plos.colors &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"#c9c9c7"</span>, <span class="st">"#0000ff"</span>, <span class="st">"#ff0000"</span>)

<span class="co"># plot the chart</span>
opar &lt;-<span class="st"> </span><span class="kw">par</span>(<span class="dt">mai =</span> <span class="kw">c</span>(<span class="fl">0.5</span>, <span class="fl">0.75</span>, <span class="fl">3.5</span>, <span class="fl">0.5</span>), <span class="dt">omi =</span> <span class="kw">c</span>(<span class="fl">0.5</span>, <span class="fl">0.5</span>, <span class="fl">1.5</span>, <span class="fl">0.5</span>), <span class="dt">mgp =</span> <span class="kw">c</span>(<span class="dv">3</span>,
    <span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="dt">fg =</span> <span class="st">"black"</span>, <span class="dt">cex.main =</span> <span class="dv">2</span>, <span class="dt">cex.lab =</span> <span class="fl">1.5</span>, <span class="dt">col =</span> plos.color,
    <span class="dt">col.main =</span> plos.color, <span class="dt">col.lab =</span> plos.color, <span class="dt">xaxs =</span> <span class="st">"i"</span>, <span class="dt">yaxs =</span> <span class="st">"i"</span>)

venn.plot &lt;-<span class="st"> </span><span class="kw">draw.triple.venn</span>(<span class="dt">area1 =</span> rows, <span class="dt">area2 =</span> summary[<span class="dv">1</span>], <span class="dt">area3 =</span> summary[<span class="dv">2</span>]/<span class="dv">2</span>,
    <span class="dt">n12 =</span> summary[<span class="dv">1</span>], <span class="dt">n23 =</span> summary[<span class="dv">3</span>], <span class="dt">n13 =</span> summary[<span class="dv">2</span>]/<span class="dv">2</span>, <span class="dt">n123 =</span> summary[<span class="dv">3</span>],
    <span class="dt">euler.d =</span> <span class="ot">TRUE</span>, <span class="dt">scaled =</span> <span class="ot">TRUE</span>, <span class="dt">fill =</span> plos.colors, <span class="dt">cex =</span> <span class="dv">2</span>, <span class="dt">fontfamily =</span> <span class="kw">rep</span>(<span class="st">"sans"</span>,
        <span class="dv">7</span>))

<span class="kw">par</span>(opar)</code></pre></div>
<figure>
<a href=""><img src="/images/2013-12-11_figure_5.svg" alt="Figure 5. PLOS Biology articles: sites of recommendation and discussion. Number of PLOS Biology research articles published up to May 20, 2013 that have been recommended by F1000Prime (red) or mentioned in Wikipedia (blue)."></a><figcaption><strong>Figure 5. PLOS Biology articles: sites of recommendation and discussion.</strong> Number of <em>PLOS Biology</em> research articles published up to May 20, 2013 that have been recommended by F1000Prime (red) or mentioned in Wikipedia (blue).</figcaption>
</figure>
<p>With the increasing availability of ALM data, there comes a growing need to provide tools that will allow the community to interrogate them. A good first step for researchers, research administrators, and others interested in looking at the metrics of a larger set of PLOS articles is the recently launched ALM Reports tool <span class="citation" data-cites="ALM2013">(“ALM Reports,” 2013)</span>. There are also a growing number of service providers, including Altmetric.com <span class="citation" data-cites="Altmetric2013">(“Altmetric.com,” 2013)</span>, ImpactStory <span class="citation" data-cites="Impactstory2013">(“ImpactStory,” 2013)</span>, and Plum Analytics <span class="citation" data-cites="Plum2013">(“Plum Analytics,” 2013)</span> that provide similar services for articles from other publishers.</p>
<p>As article-level metrics become increasingly used by publishers, funders, universities, and researchers, one of the major challenges to overcome is ensuring that standards and best practices are widely adopted and understood. The National Information Standards Organization (NISO) was recently awarded a grant by the Alfred P. Sloan Foundation to work on this <span class="citation" data-cites="NISO2013">(“NISO Alternative Assessment Metrics (Altmetrics) Project,” 2013)</span>, and PLOS is actively involved in this project. We look forward to further developing our article-level metrics and to having them adopted by other publishers, which hopefully will pave the way to their wide incorporation into research and researcher assessments.</p>
<h3 id="supporting-information">Supporting Information</h3>
<p><strong><a href="http://dx.doi.org/10.1371/journal.pbio.1001687.s001">Data S1</a>. Dataset of ALM for PLOS Biology articles used in the text, and R scripts that were used to produce figures.</strong> The data were collected on May 20, 2013 and include all <em>PLOS Biology</em> articles published up to that day. Data for F1000Prime were collected on August 15, 2013. All charts were produced with R version 3.0.0.</p>
<h2 id="references" class="references unnumbered">References</h2>
<div id="ref-ALM2013">
<p>ALM Reports. (2013). Retrieved from <a href="http://almreports.plos.org" class="uri">http://almreports.plos.org</a></p>
</div>
<div id="ref-Altmetric2013">
<p>Altmetric.com. (2013). Retrieved from <a href="http://www.altmetric.com/" class="uri">http://www.altmetric.com/</a></p>
</div>
<div id="ref-Bik2013">
<p>Bik, H. M., &amp; Goldstein, M. C. (2013). An introduction to social media for scientists. <em>PLOS Biology</em>, <em>11</em>(4), e1001535. <a href="http://doi.org/10.1371/journal.pbio.1001535">doi:10.1371/journal.pbio.1001535</a></p>
</div>
<div id="ref-bollenEtal2009">
<p>Bollen, J., Sompel, H. de, Hagberg, A., &amp; Chute, R. (2009). A Principal Component Analysis of 39 Scientific Impact Measures. <em>PLoS ONE</em>, <em>4</em>(6), e6022. <a href="http://doi.org/10.1371/journal.pone.0006022">doi:10.1371/journal.pone.0006022</a></p>
</div>
<div id="ref-Campbell2008">
<p>Campbell, P. (2008). Escape from the impact factor. <em>Ethics in Science and Environmental Politics</em>, <em>8</em>, 5–7. Journal article. <a href="http://doi.org/10.3354/esep00078">doi:10.3354/esep00078</a></p>
</div>
<div id="ref-Dalloul2010">
<p>Dalloul, R. A., Long, J. A., Zimin, A. V., Aslam, L., Beal, K., Blomberg, L. A., … Reed, K. M. (2010). Multi-platform next-generation sequencing of the domestic turkey (Meleagris gallopavo): genome assembly and analysis. <em>PLOS Biology</em>, <em>8</em>(9). <a href="http://doi.org/10.1371/journal.pbio.1000475">doi:10.1371/journal.pbio.1000475</a></p>
</div>
<div id="ref-Dickson:2010ix">
<p>Dickson, S. P., Wang, K., Krantz, I., Hakonarson, H., &amp; Goldstein, D. B. (2010). Rare variants create synthetic genome-wide associations. <em>PLOS Biology</em>, <em>8</em>(1), e1000294. <a href="http://doi.org/10.1371/journal.pbio.1000294">doi:10.1371/journal.pbio.1000294</a></p>
</div>
<div id="ref-VanEck2013">
<p>Eck, N. J. van, Waltman, L., Raan, A. F. J. van, Klautz, R. J. M., &amp; Peul, W. C. (2013). Citation analysis may severely underestimate the impact of clinical research as compared to basic research. <em>PLOS ONE</em>, <em>8</em>(4), e62395. <a href="http://doi.org/10.1371/journal.pone.0062395">doi:10.1371/journal.pone.0062395</a></p>
</div>
<div id="ref-Eysenbach2006">
<p>Eysenbach, G. (2006). Citation advantage of open access articles. <em>PLOS Biology</em>, <em>4</em>(5), e157. <a href="http://doi.org/10.1371/journal.pbio.0040157">doi:10.1371/journal.pbio.0040157</a></p>
</div>
<div id="ref-Fausto2012">
<p>Fausto, S., Machado, F. A., Bento, L. F. J., Iamarino, A., Nahas, T. R., &amp; Munger, D. S. (2012). Research blogging: indexing and registering the change in science 2.0. <em>PLOS ONE</em>, <em>7</em>(12), e50109. <a href="http://doi.org/10.1371/journal.pone.0050109">doi:10.1371/journal.pone.0050109</a></p>
</div>
<div id="ref-Floreano2010">
<p>Floreano, D., &amp; Keller, L. (2010). Evolution of adaptive behaviour in robots by means of Darwinian selection. <em>PLOS Biology</em>, <em>8</em>(1), e1000292. <a href="http://doi.org/10.1371/journal.pbio.1000292">doi:10.1371/journal.pbio.1000292</a></p>
</div>
<div id="ref-Glanzel2013">
<p>Glänzel, W., &amp; Wouters, P. (2013). The dos and don’ts in individudal level bibliometrics. Retrieved from <a href="http://de.slideshare.net/paulwouters1/issi2013-wg-pw" class="uri">http://de.slideshare.net/paulwouters1/issi2013-wg-pw</a></p>
</div>
<div id="ref-Impactstory2013">
<p>ImpactStory. (2013). Retrieved from <a href="http://impactstory.org/" class="uri">http://impactstory.org/</a></p>
</div>
<div id="ref-Lin2013">
<p>Lin, J., &amp; Fenner, M. (2013). Altmetrics in Evolution: Defining and Redefining the Ontology of Article-Level Metrics. <em>Information Standards Quarterly</em>, <em>25</em>(2), 20. <a href="http://doi.org/10.3789/isqv25no2.2013.04">doi:10.3789/isqv25no2.2013.04</a></p>
</div>
<div id="ref-Narendra:2010fw">
<p>Narendra, D. P., Jin, S. M., Tanaka, A., Suen, D.-F., Gautier, C. A., Shen, J., … Youle, R. J. (2010). PINK1 is selectively stabilized on impaired mitochondria to activate Parkin. <em>PLOS Biology</em>, <em>8</em>(1), e1000298. <a href="http://doi.org/10.1371/journal.pbio.1000298">doi:10.1371/journal.pbio.1000298</a></p>
</div>
<div id="ref-Nilsson2010">
<p>Nilsson, M. A., Churakov, G., Sommer, M., Tran, N. V., Zemann, A., Brosius, J., &amp; Schmitz, J. (2010). Tracking marsupial evolution using archaic genomic retroposon insertions. <em>PLOS Biology</em>, <em>8</em>(7), e1000436. <a href="http://doi.org/10.1371/journal.pbio.1000436">doi:10.1371/journal.pbio.1000436</a></p>
</div>
<div id="ref-NISO2013">
<p>NISO Alternative Assessment Metrics (Altmetrics) Project. (2013). Retrieved from <a href="http://www.niso.org/topics/tl/altmetrics/initiative" class="uri">http://www.niso.org/topics/tl/altmetrics/initiative</a></p>
</div>
<div id="ref-Plum2013">
<p>Plum Analytics. (2013). Retrieved from <a href="http://www.plumanalytics.com/" class="uri">http://www.plumanalytics.com/</a></p>
</div>
<div id="ref-Schekman2013">
<p>Schekman, R., &amp; Patterson, M. (2013). Reforming research assessment. <em>eLife</em>, <em>2</em>, e00855. <a href="http://doi.org/10.7554/eLife.00855">doi:10.7554/eLife.00855</a></p>
</div>
<div id="ref-Tasmanian2013">
<p>Tasmanian devil. (2013). Retrieved from <a href="http://en.wikipedia.org/wiki/Tasmanian%5Cdevil" class="uri">http://en.wikipedia.org/wiki/Tasmanian\devil</a></p>
</div>
<div id="ref-Video2012">
<p>Video Tip of the Week: Turkeys and their genomes. (2012). Retrieved from <a href="http://blog.openhelix.eu/?p=14388" class="uri">http://blog.openhelix.eu/?p=14388</a></p>
</div>
</body></html>


            <div class="license">
              
                <hr>
                <div class="toc-item">
                  <h3>Next: <a href="/2013/12/12/from-markdown-to-jats-xml-in-one-step">From Markdown to JATS XML in one Step</a></h3>
                  
                  <p>The Journal Article Tag Suite (JATS) is a NISO standard that defines a set of XML elements and attributes for tagging journal articles. JATS is not only used for fulltext content at PubMed Central (and JATS has evolved from the NLM Archiving and Interchange Tag Suite originally developed for PubMed Central), but is also increasinly used by publishers.



</p>
                </div>
              
            </div>

            <div id="disqus_thread"></div>
<script type="text/javascript">
    var disqus_developer = 1;
    var disqus_shortname = 'gobbledy'; // required: replace example with your forum shortname
    
    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">blog comments powered by <span class="logo-disqus">Disqus</span></a>

          </div>
        </div>

        <!-- COinS metadata (for citation managers like Zotero etc), goes in body text -->
<span
  class="Z3988"
  title="ctx_ver=Z39.88-2004
  &amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Adc
  &amp;rfr_id=info%3Asid%2Focoins.info%3Agenerator
  &amp;rft.title=What Can Article Level Metrics Do for You?
  &amp;rft.creator=Martin Fenner
  &amp;rft.date=2013-12-11
  &amp;rft.language=EN
  &amp;rft.rights=CC0
  &amp;rft_id=http://blog.martinfenner.org/2013/12/11/what-can-article-level-metrics-do-for-you">
</span>

        <script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-36379104-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>

      </div>
    </div>

    <!-- jQuery -->
    <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
    <script src="https://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>

    <!-- Fluidbox https://github.com/terrymun/Fluidbox -->
    <script src="/js/js-yaml.min.js"></script>
    <script src="/js/imagesloaded.pkgd.min.js"></script>
    <script src="/js/jquery.fluidbox.min.js"></script>
    <script type='text/javascript'>
      $(function () {
        $("a").fluidbox();
      });
    </script>

    <!-- Bootstrap -->
    <script src="//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
    <script src="/js/jasny-bootstrap.min.js"></script>

    <!-- MathJax -->
    <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  </body>
</html>
